setup.py

```diff
@@ -6,7 +6,7 @@
 
 INSTALL_REQUIRES = [
     'django-cms>=3.2.0',
-    'django>=1.8,<1.10',
+    'django>=1.8,<1.10.999',
```

Comment:
I think the standard is to use `<1.11` no?

Reply:
Indeed, installers such as pip and buildouts used to consider pre-release versions as matching the requirements.  Nowadays they don’t unless users opt in, so writing `< 1.11` is safe.

Reply:
Also note the new operator that matches major and minor number and allows upgrading the micro part: `django ~= 1.10, ~= 1.11`